account_invoice_ubl_peppol/README.rst

```diff
@@ -0,0 +1,105 @@
+==========================
+Account Invoice UBL PEPPOL
+==========================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fedi-lightgray.png?logo=github
+    :target: https://github.com/OCA/edi/tree/13.0/account_invoice_ubl_peppol
+    :alt: OCA/edi
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/edi-13-0/edi-13-0-account_invoice_ubl_peppol
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/226/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+With module `Account Invoice UBL`, invoices are generated according to generic UBL rules.
+
+In Europe, some countries use the PEPPOL 3.0 BIS standard as a more strict subset of UBL.
+
+With this module you can specify some or all of your invoices to be generated and validated
+according to this stricter standard.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+- Go to menu *Invoicing > Configuration > Settings > Invoicing*, under *Electronic Invoices*.
+
+- Formulate a domain for which invoices the dialect should become PEPPOL.
+  By default it is *[]*, so all UBL invoices will be PEPPOL.
+  If you want this only for Belgian partners, then you can fill here for example:
+  `[('partner_id.country_id.code', '=', 'BE')]`
+  Or you can choose to enable this only for selected partner ids.
+
+- Install OCA module 'partner_coc' and fill the field `coc_registration_number`
```

Comment:
If this is required, maybe you can add it in the manifest on the `depends`

Reply:
I'm reluctant to make this required - installations that use another field to store the COC or any other kind of PEPPOL address identifier such as the "organization number", should be able to use this module without having to install `partner_coc`.

On the other hand, if those installations dont need the partner_coc fields they can also use a custom module to hide the partner_coc fields to avoid confusion.

What do you think?

Reply:
I would prefer to add it on the depends, and special cases should make the changes (they would need to do it anyway)